kodilla-intro/src/main/java/User.java

```diff
@@ -0,0 +1,52 @@
+public class User {
+
+    String name;
```

Comment:
private

Reply:
Jak chcę zrobić "private" to mi się nie kompiluje.